chapters/ch23.md

```diff
@@ -1,36 +1,35 @@
 # 23. Addressing Bias and Variance
--> 
-
+->
+Giải quyết Độ chệch và Phương sai
```

Comment:
Addressing mình thấy không thể dịch là "giải quyết" được đâu. Có chăng là **định vị** thôi.

Reply:
Mình nghĩ  nó vẫn mang ý nghĩa để giải quyết cái gì đó. Ví dụ như câu "This problem has been addressed".

Một cách khác có thể dịch là "Xử lý Độ chệch và Phương sai"
@duythanhvn  bạn thấy sao ?